cirro/cli/cli.py

```diff
@@ -59,8 +59,8 @@ def download(**kwargs):
               default='')
 @click.option('--project',
               help='Name or ID of the project')
-@click.option('--process',
-              help='Name or ID of the ingest process')
+@click.option('--data-type', '--process',
+              help='Name or ID of the data type (ingest process)')
```

Comment:
Suggest indicating in the help text that the `--process` option is deprecated and will be removed in a future version.